src/main/java/com/google/sps/data/Game.java

```diff
@@ -0,0 +1,117 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.ArrayList;
+
+// Represents the data of a single Game
+public class Game {
+    private final String gameID;
+    private final String gameName;
+    private final String gameDescription;
+    private final int numStages;
+    private final ArrayList<String> stages;
+    private final int numTimesPlayed;
+    private final int numTimesFinished;
+    private final int numStarVotes;
+    private final int totalStars;
+    private final int numDifficultyVotes;
+    private final int totalDifficulty;
+
+    public static class Builder {
+        // Required parameters
+        private final String gameID;
+        private final String gameName;
+
+        // Optional paramters - initialized to default values
+        private String gameDescription = null;
```

Comment:
Wouldn't it be better to set it to an empty string rather than null? This way if you print the gameDescription to the page or use it there is less of a chance of null pointer exceptions.

Reply:
I think null is the better choice here because this way, there is a clear distinction between the servlet having set no description and the description being an empty string.